packages/yoga/src/Tag/web/Informative.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { func, oneOfType, oneOf, bool, node } from 'prop-types';
+import { hexToRgb } from '@gympass/yoga-common';
+
+import Tag from './Tag';
+
+const Informative = styled(Tag)`
+  justify-content: center;
+  align-items: center;
+
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        colors: {
+          text,
+          feedback: { [variant]: color = { light: text.secondary } },
+        },
+        components: { tag },
+      },
+    },
+  }) => `
+    background-color: ${hexToRgb(color.light)};
+    color: ${text.primary};
+    border-radius: ${tag.border.radius}px;
+    border-color: ${color.light};
+
+    font-size: ${tag.font.size}px;
+    font-weight: ${tag.font.weight};
+
+    svg {
+      margin-right: ${tag.icon.margin.right}px;
+    }
+  `}
+`;
+
+const TagInformative = ({
+  children,
+  icon: Icon,
+  theme: {
+    yoga: {
+      colors: { primary, color = primary },
```

Comment:
```suggestion
      colors: { primary },
```

We don't need this `color` alias, right? 
Also, it should be better to check if the color is whether `primary` or `text.primary`